line-sdk/src/main/java/com/linecorp/linesdk/auth/LineLoginResult.java

```diff
@@ -147,6 +165,16 @@ public LineProfile getLineProfile() {
         return lineProfile;
     }
 
+    /**
+     * Gets the ID token raw string.
+     *
+     * @return A ID token raw string.
+     */
+    @Nullable
+    public String getLineIdTokenRawStr() {
```

Comment:
please use full name `getLineIdTokenRawString()`

Reply:
removed this